app/models/katello/glue/pulp/repo.rb

```diff
@@ -255,11 +255,13 @@ def distribution_ids
       end
 
       def package_group_count
-        self.pulp_repo_facts[:content_unit_counts][:package_group] || 0
+        content_unit_counts = self.pulp_repo_facts[:content_unit_counts]
+        content_unit_counts ? content_unit_counts[:package_group] : 0
       end
 
       def puppet_module_count
-        self.pulp_repo_facts[:content_unit_counts][:puppet_module] || 0
+        content_unit_counts = self.pulp_repo_facts[:content_unit_counts]
+        content_unit_counts ? content_unit_counts[:puppet_module] : 0
       end
```

Comment:
Yeah, the error referenced in the bug:  `   undefined method '[]' for nil:NilClass`.